clouddriver-aws/src/main/java/com/netflix/spinnaker/clouddriver/aws/lifecycle/InstanceTerminationLifecycleWorker.java

```diff
@@ -227,13 +228,22 @@ private boolean updateInstanceStatus(Eureka eureka, String app, String instanceI
       try {
         eureka.updateInstanceStatus(app, instanceId, DiscoveryStatus.OUT_OF_SERVICE.getValue());
         return true;
-      } catch (RetrofitError e) {
-        final String recoverableMessage =
```

Comment:
Leaving this in minimizes diffs, and reduces duplication, yes?

Reply:
yes

Reply:
will add it back